lib/agora/agora_dispatcher.ex

```diff
@@ -0,0 +1,91 @@
+defmodule Membrane.Agora.Dispatcher do
+  @moduledoc false
+
+  use Membrane.Filter
+
+  require Logger
+  require Membrane.Logger
+
+  def_input_pad(:input,
+    flow_control: :auto,
+    accepted_format: _any
+  )
+
+  def_output_pad(:output,
+    flow_control: :auto,
+    accepted_format: _any,
+    availability: :on_request
+  )
```

Comment:
- auto is the default
- add accepted format

Reply:
done